src/index.js

```diff
@@ -27,6 +27,9 @@ async function publishConfig(params) {
     params.__ow_logger,
     params.indexconfig,
     params.algoliaappid,
+    params.wskAuth,
+    params.wskHost,
```

Comment:
this should default to `process.env.__OW_API_HOST`

Reply:
^ Right! Good call, i'll fix this asap tomorrow morning

Reply:
I think this will happen in the publish command in helix-cli!

Reply:
https://github.com/adobe/helix-cli/blob/318770004ea81dd88bc825b73fc124343f24efd0/src/remotepublish.cmd.js#L285-L326